db/migrate/20231208053131_create_projects.rb

```diff
@@ -0,0 +1,12 @@
+class CreateProjects < ActiveRecord::Migration[7.0]
+  def change
+    create_table :projects do |t|
+      t.string :name
+      t.text :description
+      t.text :image_url
+      t.date :starting_date
```

Comment:
would recommend using `start_date` rather than `starting`
also, since we will probably want to cycle through active projects and some might be already finished, it would be good to have an end date as well, despite not being defined from the project's beginning